camille/process/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import atm_stb, delta_temp, fft, mooring_fatigue, rolling
```

Comment:
In this one I think we should import the process functions
```suggestion
from .atm_stb import process as atm_stb
from .delta_temp import process as delta_temp
from .fft import process as fft
from .mooring_fatigue import process as mooring_fatigue
from .rolling import process as rolling
```

Reply:
So you want to do: `import camille` and  `camille.process.fft()`?

Reply:
Yes, I think it would be an improvement